Can't build 0.6.1

Version 0.6.1 on Fedora Rawhide, rust 1.35.0:

```
BUILDSTDERR:    Compiling sd v0.6.1 (/builddir/build/BUILD/sd-0.6.1)
BUILDSTDERR:      Running `/usr/bin/rustc --edition=2018 --crate-name sd src/main.rs --color never --emit=dep-info,link -C opt-level=3 --test -C metadata=db7c97756d2f4430 -C extra-filename=-db7c97756d2f4430 --out-dir /builddir/build/BUILD/sd-0.6.1/target/release/deps -L dependency=/builddir/build/BUILD/sd-0.6.1/target/release/deps --extern memmap=/builddir/build/BUILD/sd-0.6.1/target/release/deps/libmemmap-b3f2069516387a5d.rlib --extern rayon=/builddir/build/BUILD/sd-0.6.1/target/release/deps/librayon-a99e4203c6332a21.rlib --extern regex=/builddir/build/BUILD/sd-0.6.1/target/release/deps/libregex-98ffbc1cf6224c5b.rlib --extern structopt=/builddir/build/BUILD/sd-0.6.1/target/release/deps/libstructopt-01b05acb492cc451.rlib --extern tempfile=/builddir/build/BUILD/sd-0.6.1/target/release/deps/libtempfile-b901b63516f8d697.rlib --extern unescape=/builddir/build/BUILD/sd-0.6.1/target/release/deps/libunescape-fd1a3ef56a6eb871.rlib -Copt-level=3 -Cdebuginfo=2 -Clink-arg=-Wl,-z,relro,-z,now -Ccodegen-units=1`
BUILDSTDERR: error[E0308]: mismatched types
BUILDSTDERR:    --> src/input.rs:181:58
BUILDSTDERR:     |
BUILDSTDERR: 181 |         assert_eq!(std::str::from_utf8(&replacer.replace(src)), Ok(target));
BUILDSTDERR:     |                                                          ^^^ expected slice, found str
BUILDSTDERR:     |
BUILDSTDERR:     = note: expected type `&[u8]`
BUILDSTDERR:                found type `&'static str`
BUILDSTDERR: error: aborting due to previous error
BUILDSTDERR: For more information about this error, try `rustc --explain E0308`.
BUILDSTDERR: error: Could not compile `sd`.

```

--- a/src/input.rs
+++ b/src/input.rs
@@ -178,7 +178,10 @@ mod tests {
             flags.map(ToOwned::to_owned),
         )
         .unwrap();
-        assert_eq!(std::str::from_utf8(&replacer.replace(src)), Ok(target));
+        assert_eq!(
+            std::str::from_utf8(&replacer.replace(src.as_bytes())),
+            Ok(target)
+        );
     }
 
     #[test]

